wagtail/admin/tests/test_forms.py

```diff
@@ -18,6 +23,31 @@ class CustomPasswordResetForm(PasswordResetForm):
     captcha = CharField(label="Captcha", help_text="should be in extra_fields()")
 
 
+class CustomImageField(ImageField):
+    def to_python(self, data):
+        avatar = data
+        avatar.image = Image.open(avatar)
+        avatar.content_type = "image/png"
+        avatar.seek(0)
+        return avatar
+
+
+class CustomAvatarPreferenceForm(AvatarPreferencesForm):
```

Comment:
Question: I'm not sure I see why a custom form is necessary for tests. Can the real one be used?

Reply:
If i use the real one, i would be able to get the image being processed to test if the size was changed